tests/Unit/Http/Action/Pages/HomePageActionTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Unit\Http\Action\Pages;
+
+use OpenCFP\Http\Action\Pages\HomePageAction;
+use OpenCFP\Test\Unit\Http\Action\AbstractActionTestCase;
+use Symfony\Component\HttpFoundation;
+
+/**
+ * @covers \OpenCFP\Http\Action\Pages\HomePageAction
+ */
+final class HomePageActionTest extends AbstractActionTestCase
+{
+    public function testItReturnsTheCorrectContentIfNoSubmissionCountNeedsToBeShown()
+    {
+        $content = $this->faker()->text();
+        $twig    = $this->createTwigMock();
+
+        $twig
+            ->expects($this->once())
+            ->method('render')
+            ->with(
+                $this->identicalTo('home.twig'),
+                $this->identicalTo(['number_of_talks' => ''])
+            )
+            ->willReturn($content);
+        $action   = new HomePageAction($twig, false);
+        $response = $action();
+        $this->assertInstanceOf(HttpFoundation\Response::class, $response);
```

Comment:
What do you think about adding some vertical whitespace here and elsewhere, for example:

```diff
diff --git a/tests/Unit/Http/Action/Pages/HomePageActionTest.php b/tests/Unit/Http/Action/Pages/HomePageActionTest.php
index f059594f..09ba426a 100644
--- a/tests/Unit/Http/Action/Pages/HomePageActionTest.php
+++ b/tests/Unit/Http/Action/Pages/HomePageActionTest.php
@@ -25,7 +25,8 @@ final class HomePageActionTest extends AbstractActionTestCase
     public function testItReturnsTheCorrectContentIfNoSubmissionCountNeedsToBeShown()
     {
         $content = $this->faker()->text();
-        $twig    = $this->createTwigMock();
+
+        $twig = $this->createTwigMock();

         $twig
             ->expects($this->once())
@@ -35,8 +36,11 @@ final class HomePageActionTest extends AbstractActionTestCase
                 $this->identicalTo(['number_of_talks' => ''])
             )
             ->willReturn($content);
-        $action   = new HomePageAction($twig, false);
+
+        $action = new HomePageAction($twig, false);
+
         $response = $action();
+
         $this->assertInstanceOf(HttpFoundation\Response::class, $response);
         $this->assertContains($content, $response->getContent());
         $this->assertSame(HttpFoundation\Response::HTTP_OK, $response->getStatusCode());
```